__tests__/app.test.js

```diff
@@ -60,6 +61,37 @@ describe("GET Endpoints", () => {
     });
   });
 
+  describe("/api/articles/:article_id/comments (06)", () => {
+    //returns comments for an article selected by id
+    // Has: comment_id, votes, created_at, author, body, article_id. 
+    it("200: Returns object with array containing all comments belonging to a specific article id", () => {
+      return request(app)
+        .get("/api/articles/3/comments") // currently 2 comments
```

Comment:
What if the user entered api/articles/banana? With your current code, would this return the correct error code.